frontend/modules/common/config.js

```diff
@@ -1,19 +1,19 @@
+var apiHost = "http://localhost:8000"
+var apiUrl = apiHost + '/api/v1'
 
 export var serverURLs = {
-  auth_token: '/api/v1/accounts/obtain-auth-token/',
-  browse: '/api/v1/recipe/recipes/?format=json&fields=id,title,pub_date,rating,photo_thumbnail,info',
-  mini_browse: '/api/v1/recipe/mini-browse/?format=json',
```

Comment:
This endpoint is needed. Its used to return a small (or large) random subset or recipes to be used on the home page and when viewing individual recipes.